pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -309,6 +310,9 @@ spec:
               # OPENEBS_IO_CSTOR_ID env has UID of cStorPool CR.
             - name: OPENEBS_IO_CSTOR_ID
               value: {{.TaskResult.putcstorpoolcr.objectUID}}
+              # OPENEBS_NAMESPACE env has namespace of.
+            - name: OPENEBS_NAMESPACE
```

Comment:
how upgrades can handle this?

Reply:
While upgrading we need to patch this `OPENEBS_NAMESPACE` env.